test/test_kvtree_write_gather.c

```diff
@@ -1,102 +1,125 @@
 #include "test_kvtree.h"
 #include "test_kvtree_util.h"
-#include "kvtree_mpi.h"
-#include <stdio.h>
+#include "mpi.h"
+#include "stdio.h"
```

Comment:
Both `mpi.h` and `stdio.h` are system libraries and should be inside angle brackets: < >